projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss

```diff
@@ -43,18 +43,20 @@
 
   .open {
     animation: openDD var(--wvr-dropdown-menu-animation-speed) ease-in-out;
+    z-index: 1200;
+    background: var(--wvr-white);
```

Comment:
This should be `var(--wvr-dropdown-menu-background)`

Reply:
dropdown background color changed to var(--wvr-dropdown-menu-background)